### ISSUE TITLE
Cannot open follow requests in UI

### ISSUE THREAD

**Issue by vmstan** [mastodon/mastodon#27535]
### Steps to reproduce the problem

1. Visit [/follow_requests](https://vmst.io/follow_requests)
2. Get error

![image](https://github.com/mastodon/mastodon/assets/3002053/150e6703-41ed-434b-b1d6-936eb4c55cf1)

```
TypeError: Cannot call a class constructor without |new|
l (::)
Xh (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:3340:8)
ug (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:6433:13)
Vk (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:6127:13)
Uk (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:6121:25)
Jk (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:6107:4)
Ok (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5830:9)
Qg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5767:53)
h (webpack:///node_modules/scheduler/cjs/scheduler.production.min.js:94:18)
R (webpack:///node_modules/scheduler/cjs/scheduler.production.min.js:125:10)
run (webpack:///node_modules/setimmediate/setImmediate.js:39:8)
setTimeout (webpack:///node_modules/setimmediate/setImmediate.js:67:14)
event.data.indexOf (webpack:///node_modules/setimmediate/setImmediate.js:104:8)
```

### Expected behaviour

See follow requests

### Actual behaviour

Error above

### Mastodon instance

vmst.io

### Mastodon version

main

### Browser name and version

All browsers

### Operating system

All OS

### Technical details

4.3.0 at commit 15182d1

**Comment by renchap** [comment]
Should be fixed by #27539 and #27540

**Comment by vmstan** [comment]
Confirmed.
